database.sql

```diff
@@ -1,6 +1,6 @@
 -- ------------------------------------------
 -- Friendica 2021.09-dev (Siberian Iris)
--- DB_UPDATE_VERSION 1432
+-- DB_UPDATE_VERSION 1433
```

Comment:
If my current PR #10593 is merged first, you will have to manually increate the database version. The system will not show a conflict here.

Reply:
addressed here: https://github.com/friendica/friendica/pull/10582/commits/92d532db6cac4a52840a003a3e62e3a4064bed51
thx for the hint!